apps/pie-microsite/src/content/pages/all-about-pie/our-vision.md

```diff
@@ -0,0 +1,43 @@
+---
+eleventyNavigation:
+    key: Our vision
+    parent: All about PIE
+title: Our vision
+description: Sustain a JET design system which empowers us all to create consistent, effective and inclusive products for our users.
+---
+
+## Mission
+
+To maintain and evolve our single source of truth design system and product components for our end users and all JET’s product-focused teams
+
+---
+
+## Principles
+
+What success looks like:
+
+### Empowered
+
+- Everyone at JET contributes to PIE and understands how our decisions are made
+- Anyone can give us feedback easily and we will listen
+- We welcome input at every stage from design to development
+
+### Consistent
+
+- We work methodically to ensure our design system is clear and user-friendly
+- We maximise our flexibility by adopting a modular approach
+- We give regular, timely progress updates to JET’s product-focused teams
+
+### Inclusive
+
+- Our output is for everyone: combining accessibility with the best design practices
```

Comment:
probably one for design actually but should these bullet points have full-stops?

Reply:
yep just share it with them. 💯

Reply:
Just amended the designs to add full stops, thanks!